server/predictions.py

```diff
@@ -4,9 +4,17 @@
 from firebase_admin import credentials
 from firebase_admin import firestore
 import server.auth as auth
-from main import getYml, easternTime
 import datetime
 
+def getYml(filepath):
+    with open(filepath, encoding='utf-8') as infile:
+        fileMap = yaml.safe_load(infile)
+        return fileMap
+
+def easternTime():
+    """ Returns a timezone object representing EST """
+    return datetime.timezone(-datetime.timedelta(hours=5))
+

```

Comment:
apparently if you import these functions from main, the `init_*_firebase` functions get called again, which throws an error